Add info to README about container images. (#42)
esp-rs/rust-build: README.md

### Rust with Podman or Docker

> Alternatively you might build the project in the container where the image already contains pre-installed Rust and ESP-IDF.

Reviewer: ```suggestion
Alternatively, you might build the project in the container where the image already contains pre-installed Rust and ESP-IDF.
```

There are two different images published to Dockerhub: 
- [idf-rust](https://hub.docker.com/r/espressif/idf-rust) - contains only the toolchain.

> - [idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes example code from https://github.com/espressif/rust-esp32-example and is about 1GB larger in size. 

Reviewer: ```suggestion
- [idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes examples code from [rust-esp32-example](https://github.com/espressif/rust-esp32-example) and the second one is [rust-esp32-std-demo](https://github.com/ivmarkov/rust-esp32-std-demo) and is about 10GB larger in size. 
```
`idf-rust-examples` consists of 2 examples, first is [rust-esp32-example](https://github.com/espressif/rust-esp32-example) and the second one is [rust-esp32-std-demo](https://github.com/ivmarkov/rust-esp32-std-demo). Image size is than about 10GB.

Author: Ah, good point on the two examples.
Re: size -> I was referring to the size according to Dockerhub, where the image for `idf-rust-examples` is 3.4GB compared to 2.5GB for `idf-rust`. Should we just leave that part out?

Reviewer: I see, I was referring to the downloaded size. I think we can leave that part out :)

> - ![idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes example code from https://github.com/espressif/rust-esp32-example and is about 1GB larger in size. 

Reviewer 2: Links are not working properly due to the `!` at the start, they are being treated as images.

Author: Oops, dumb mistake 😬

docker run -it espressif/idf-rust-examples
```

> If you are using the `idf-rust-examples` image, instructions will displayed on the screen.

Reviewer: ```suggestion
If you are using the `idf-rust-examples` image, instructions will be displayed on the screen.
```
instructions will `be` displayed.
